bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -34,10 +34,14 @@ class ControlStateManager : public ResPagesClient<ControlStateManager,
   ControlStateManager(const ControlStateManager&) = delete;
   ~ControlStateManager() {}
 
+  void disable() { enabled_ = false; }
```

Comment:
Your commit message doesn't describe why we can/want to "enable"/"disable" the ControlStateManager. What is its purpose? Are you using it as a feature flag?
A comment would help that explains in what scenario you call either.

Reply:
This is for cases when we don't want to use this feature (for example, in some of the simple tests)